common/compositor/va/varenderer.cpp

```diff
@@ -357,4 +296,39 @@ int VARenderer::DrmFormatToRTFormat(int format) {
   return 0;
 }
 
+bool VARenderer::CreateContext() {
+  DestroyContext();
+
+  VAConfigAttrib config_attrib;
+  config_attrib.type = VAConfigAttribRTFormat;
+  config_attrib.value = render_target_format_;
+  VAStatus ret =
+      vaCreateConfig(va_display_, VAProfileNone, VAEntrypointVideoProc,
+                     &config_attrib, 1, &va_config_);
+  if (ret != VA_STATUS_SUCCESS) {
+    ETRACE("Create VA Config failed\n");
+    return false;
+  }
+
+  // These parameters are not used in vaCreateContext so just set them to dummy
+  // values
+  int width = 1920;
```

Comment:
How about 1 and 1.

Reply:
as the comment said, just set dummy values as the driver impl actually doesn't use these values.

Reply:
Yes usually dummy values are 1 :)